tomcat/src/main/java/org/apache/coyote/http11/HttpHeader.java

```diff
@@ -42,8 +65,7 @@ public void add(String name, String value) {
     }
 
     public String buildMessage() {
-        return headers.keySet().stream()
-                .map(key -> key + DELIMITER + headers.get(key))
+        return headers.keySet().stream().map(key -> key + DELIMITER + " " + headers.get(key))
```

Comment:
[반영하지 않으셔도 됩니다]
`keySet`을 사용하셨군요 😄 `entrySet`을 사용해보는건 어떻게 생각하시나요?

```suggestion
        String RESPONSE_HEADER_FORMAT = "%s: %s";
        // ... 중략 ...
        return headers.entrySet().stream()
                .map(entry -> RESPONSE_HEADER_FORMAT.formatted(entry.getKey(), entry.getValue()))
                .collect(Collectors.joining(CRLF));
```

Reply:
hoxy genius?